destreamer.ts

```diff
@@ -51,6 +51,13 @@ const argv = yargs.options({
         default: false,
         demandOption: false
     },
+    noThumbnails: {
+        alias: 'nthumb',

```

Comment:
I wouldn't create an alias for such an option

Reply:
why not? 
full option is lengthy and we are all lazy :P

Reply:
I think the cooler name for that would be `--no-art`, but let's rename when we switch to --kebab-case for all CLI args.